### PR TITLE
fix unit test breakage by adding seed method to util/rand

### PR DESCRIPTION
**Pull request by mikedanese**
@vishh @thockin

### CODE REVIEW

**Comment by vishh** [comment]
LGTM

**Comment by vishh** [comment]
Thanks for the quick fix @mikedanese

**Comment by vishh** [comment]
@k8s-bot: retest this please

**Comment by vishh** [comment]
@k8s-bot: test this please

**Comment by mikedanese** [comment]
@k8s-bot  ok to test

**Comment by vishh** [comment]
@mikedanese: Shippable has been flaky with this PR.

**Comment by mikedanese** [comment]
@k8s-bot ok to test

File: pkg/serviceaccount/tokens_controller_test.go [hunk 1]
```diff
@@ -372,7 +372,7 @@ func TestTokenCreation(t *testing.T) {
 	for k, tc := range testcases {
 
 		// Re-seed to reset name generation
-		rand.Seed(1)
+		utilrand.Seed(1)
```
Comment (thockin) [inline pkg/serviceaccount/tokens_controller_test.go]: I really dislike that some random test is dependent on the seed of some distant rng library.  Shouldn't this test be written to be independent of the random portion?

**Comment by mikedanese** [comment]
@k8s-bot ok to test

File: pkg/serviceaccount/tokens_controller_test.go (hunk 1 above)
Comment (thockin) [inline pkg/serviceaccount/tokens_controller_test.go]: I withdraw my comment.  Fixing it is more effort that it is worth.  In an ideal world this coupling would be looser, but it's probably not worth the energy.

Something like api.NameGenerator.Seed(1) which was an instance of our own Rand object from this lib, which was used for all name generation.  That way it was clear what the test was asserting.

**Comment by thockin** [comment]
LGTM

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Comment by wojtek-t** [comment]
Test failure seems to be unrelated:
Kubernetes e2e suite.Monitoring should verify monitoring pods and all cluster nodes are available on influxdb using heapster.

Rerunning e2e tests

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Comment by wojtek-t** [comment]
e2e test flake:
Kubernetes e2e suite.Services should serve multiport endpoints from pods

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Pull request merged**